javatests/arcs/android/e2e/testapp/TestActivity.kt

```diff
@@ -432,6 +459,9 @@ class TestActivity : AppCompatActivity() {
         result2 = result
         resultView1.text = "1: $result1"
         resultView2.text = "2: $result2"
+        if (devToolsBound) {
+            devToolsService?.send(result);
```

Comment:
if presence of `devToolsService` would indicate whether it's bound, then you could probably remove `?.` and have a regular `.`.

i.e.
```
        if (devToolsService != null) {
            devToolsService.send(result);
        }
```